packages/grid/_modules_/grid/models/params/gridRowParams.ts

```diff
@@ -3,17 +3,17 @@ import { GridRowId, GridRowModel } from '../gridRows';
 import type { GridColumns } from '../colDef';
 
 /**
- * Object passed as parameter in the column [[GridColDef]] cell renderer.
+ * Object passed as parameter in the row callbacks.
  */
-export interface GridRowParams {
+export interface GridRowParams<RowDataType = any> {
```

Comment:
@m4theushw do we support row that are not objects ?

If not, we should limit `RowDataType` 

```tsx
type GridRowData = { [key: string]: any }

export interface GridRowParams<RowDataType extends GridRowData = GridRowData> {
```

Same in other usages of `row`

Reply:
I limited row as only objects.